src/main/resources/templates/fragments/screens/screenWithOneInput.html

```diff
@@ -42,3 +43,20 @@
   </body>
   </html>
 </th:block>
+
+<th:block
+  th:replace="~{fragments/screens/screenWithOneInput ::
+  screenWithOneInput(
+    title=#{},
+    header=#{},
+    inputName='',
+    subtext=#{},
+    formAction=${formAction},
+    inputContent=~{::inputContent})}">
+  <th:block th:ref="inputContent">
+    <!-- Be sure to have `ariaLabel='header'` to label the input with the header -->
+    <th:block th:replace="~{fragments/inputs/text ::
+      text(inputName='',
+      ariaLabel='header')}"/>
+  </th:block>
+</th:block>
```

Comment:
Is this meant to be here? If so why?

Reply:
The fragment in a fragment? I was wondering about that, too.

Reply:
Good catch, not meant to be there